app/assets/stylesheets/navbar.scss

```diff
@@ -9,22 +9,27 @@
 
 .navbar-nav li a {
   color: white;
-  font-family: 'Open Sans', sans-serif;
+  font-family: 'Questrial', sans-serif;
```

Comment:
Same for the default font-family. Set it site wide and overwrite it for those specific items where you need another font.